contracts/token/RarePizzasBox.sol

```diff
@@ -29,18 +29,30 @@ contract RarePizzasBox is
     using CountersUpgradeable for CountersUpgradeable.Counter;
     using SafeMathUpgradeable for uint256;
 
-    uint256 private constant MAX_TOKEN_SUPPLY = 10000;
-    uint256 private constant MAX_MINTABLE_SUPPLY = 1250; // TODO: verify
+    // V1 Variables (do not modify this section when upgrading)
+
+    // 2021-03-14:15h::9m::26s
+    uint256 public _public_sale_start_timestamp = 1615734566;
 
-    CountersUpgradeable.Counter private _minted_pizza_count;
-    CountersUpgradeable.Counter private _whole_pizza_count;
+    uint256 public constant MAX_TOKEN_SUPPLY = 10000;
+    uint256 public constant MAX_MINTABLE_SUPPLY = 1250;
+    uint256 public constant MAX_PURCHASABLE_SUPPLY = 8750;
 
-    // TODO:
-    string private _boxMetadataUri =
+    string public _boxMetadataUri =
         "https://ipfs.io/ipfs/some/path/to/box/metadata";
 
-    function initialize() public initializer {
+    CountersUpgradeable.Counter public _minted_pizza_count;
+    CountersUpgradeable.Counter public _purchased_pizza_count;
+
+    // END V1 Variables
+
+    function initialize(address mintTo, uint8 count) public initializer {
```

Comment:
Should we check this address?

Reply:
what do you mean check it?  this function gets called on contract deployment so I am certainly assuming that the deployer knows what they are doing.  i was actually considering removing the mint functionality and just forcing it to be called separately